plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3852,6 +3856,10 @@ blockly.errors.disabled_block_type.remove=Disabled block type {0}. Remove this b
 blockly.errors.unknown_block_type.remove=Unknown block type {0}. Remove this block!
 blockly.errors.cancel_event.null=Failed to load selected external trigger.
 blockly.errors.cancel_event.not_cancellable=The selected global trigger is not cancellable.
+blockly.errors.event_parameter_set.no_value=Global trigger parameters can't be set because value is not defined.
```

Comment:
Please, don't use single ' in translation files because they need to be escaped. This is how currently those texts look like:

![image](https://github.com/user-attachments/assets/98118e0a-819a-4384-b12b-cec931d74cfe)